src/components/trail.style.js

```diff
@@ -1,11 +1,43 @@
 export default {
   container: {
-    flex: 1
+    flex: 1,
+  },
+  mapView: {
+      flex: 1,
+      
+      top: 0,
```

Comment:
i don't think you ned the top, left, bottom, and right values here. try removing them and see if they make any difference. typically those are only used with absolute or relative position elements